src/main/resources/templates/generators/overviewTags.vm

```diff
@@ -5,7 +5,7 @@
   #includeTitle("Tags Overview")
 
   <script type="text/javascript">
-  #parse("/templates/js/tags-chart.js")
+  #parse("/templates/js/tags-chart.js.vm")
```

Comment:
if you want to change it to vm then apply this change for all files - after your change some JavaScript charts are in js files, others in vm